### DISCUSSION TITLE
AqualinkD Devices in MQTT for HASS?

### DISCUSSION THREAD

**Discussion by parchmaf** [aqualinkd/AqualinkD#283] [Announcements]
Is it possible to have AqualinkD devices show up as devices in MQTT within Homeassistant?  For example, the Ring implementation with Homeassistant is not so great so I installed Ring-MQTT which puts all my devices in HASS using the MQTT implementation within HASS.  It works much better.  The Jandy implementation within HASS seems to be down more than up so my thought was maybe AqualinkD could do the same thing.  AqualinkD and Ring-MQTT are using the same Mosquitto server and even though this MQTT server is configured in HASS, I only see ring devices.  I do not see the AqualinkD devices.  Not sure if this is possible but it would be really nice if it was.  Attached is a screenshot of my MQTT devices within Home Assistant for reference.
<img width="1754" alt="Screenshot 2024-04-24 at 5 38 58 PM" src="https://github.com/sfeakes/AqualinkD/assets/73364970/1d165c69-ed3e-49ed-8ee7-f6dae48ac2af">

**Comment by sfeakes** [comment]
To get AqualinkD into HASS via MQTT, you simply need to use the example file in this repo and modify it for your specific setup, then load it into HASS.

Having AqualinkD publish the config via MQTT in HASS format is on my list of todo, but low ish on priority, since you easily modify the file. 

Few links that will help
https://github.com/sfeakes/AqualinkD/wiki#HASSIO
https://github.com/sfeakes/AqualinkD/issues/78
